src/commands.js

```diff
@@ -28,25 +28,28 @@ commands.set('deposit', {
 
     try {
       // Submit the transfer transaction
-      const Avail = await AvailApi.create();
       const dest = interaction.options.get('address', true).value;
-      await Avail.transfer({ dest, amount: 1, onResult: (result) => {
+      console.log(`Received deposit request for ${dest}`);
+      const api = await createApi('testnet');
+      const mnemonic = process.env.SEED_PHRASE;
+      await transfer({api, mnemonic, dest, amount: 1, onResult: (result) => {
         if (result.status.isInBlock) {
           const blockHash = result.status.asInBlock;
           const link = 'https://testnet.avail.tools/#/explorer/query/' + blockHash;
           interaction.followUp({
             content: `Status: Complete
-Amount:  1 AVL
-Txn Hash: ${result.txHash}
-Block Hash: ${blockHash}
-🌐 ${hyperlink('View in explorer', link)}`
+            Amount:  1 AVL
+            Txn Hash: ${result.txHash}
+            Block Hash: ${blockHash}
+            🌐 ${hyperlink('View in explorer', link)}`
           });
         }
       }});
+      console.log(`transferred 1 AVL to ${dest}`)
     } catch (error) {
       console.error(error);
       interaction.followUp({
-        content: `There was a problem transferring to ${dest}. Kindly report to the Avail Team.`,
+        content: `There was a problem transferring. Kindly report to the Avail Team.`,
```

Comment:
More grammatical: "There was a problem with the transfer. Kindly report to the Avail Team."